### PR TITLE
Update documentation (Development Environment section)

### LINKED ISSUE #4072: Update CONTRIBUTING.md guide to run debug version of CVAT
### My actions before raising this issue
* [x]  Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
* [x]  Searched [past issues](/issues)

### Expected Behaviour
Sign in successfully to CVAT.
### Current Behaviour
I am getting the following error: 
Screenshot from 2021-12-23 08-48-15
### Steps to Reproduce (for bugs)

1. I had CVAT worked well with the latest commits in the development environment.
2. I pulled the last commit.
3. I ran these commands in order to update CVAT : 
    - ```pip install -r cvat/requirements/development.txt```
    - ```python manage.py migrate```
    - ```python manage.py collectstatic```
    - ```npm ci```
4. I got the same error that I am getting now so I completely reinstalled CVAT using [these](https://openvinotoolkit.github.io/cvat/docs/contributing/development-environment/) steps.
### Your Environment
Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu 18.04

Here are the logs from VScode console:
[server-chrome-logs.txt](https://github.com/openvinotoolkit/cvat/files/7767333/server-chrome-logs.txt)
[server-django-logs.txt](https://github.com/openvinotoolkit/cvat/files/7767336/server-django-logs.txt)

### PR DESCRIPTION
**Pull request by AmitN1212**
resolved #4072 #3844 

### Motivation and context

Make the installation instructions of the development environment much clearer, accurate and understandable
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by AmitN1212** [comment]
@bsekachev I changed only a documentation file, so how didn't it pass the tests?

File: site/content/en/docs/contributing/development-environment.md [hunk 1]
```diff
@@ -73,25 +80,36 @@ description: 'Installing a development environment for different operating syste
   >
   > Read this article [Node Sass does not yet support your current environment](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
 
-- Open new terminal (Ctrl + Shift + T), run Visual Studio Code from the virtual environment
+- Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker-Compose](https://docs.docker.com/compose/install/)
```
Comment (ActiveChooN) [inline site/content/en/docs/contributing/development-environment.md:85]: Or docker-desktop for Mac & Windows

File: site/content/en/docs/contributing/development-environment.md [hunk 2]
```diff
@@ -73,25 +80,36 @@ description: 'Installing a development environment for different operating syste
...
+  ```sh
+  source .env/bin/activate && code
+  ```
+
+- Inside VScode, Open CVAT root dir
 
-- Reload Visual Studio Code from virtual environment
+- Select `server: debug` configuration and run it (F5) to run REST server and its workers
+- Make sure that ```Uncaught Exceptions``` option under breakpoints section is unchecked
+- If you choose to run CVAT in localhost: Select `server: chrome` configuration and run it (F5) to open CVAT in Chrome
```
Comment (ActiveChooN) [inline site/content/en/docs/contributing/development-environment.md:113]: There is also `ui.js: debug` configuration with npm background task

File: site/content/en/docs/contributing/development-environment.md (hunk 1 above)
Comment (AmitN1212) [reply site/content/en/docs/contributing/development-environment.md:85]: But below, in Note for windows users, it says to install WSL and then follow all the commands for Ubuntu... So why would we need to install Docker for Windows?
For Mac: Maybe it is better to change the link for [this](https://docs.docker.com/engine/install/).

File: site/content/en/docs/contributing/development-environment.md (hunk 2 above)
Comment (AmitN1212) [reply site/content/en/docs/contributing/development-environment.md:113]: > There is also `ui.js: debug` configuration with npm background task

It does not work for me. Maybe I am trying to do something wrong.

**Review by nmanovic — APPROVED** [review]
@AmitN1212 , thanks for the contribution.

**Pull request merged**